data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,115 @@
-
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "pattern": "^https:\\/\\/hdl\\.handle\\.net\\/20\\.5000\\.1025\\/.{3}-.{3}-.{3}$",
+      "examples": ["https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"],
+      "description": "Unique identifier for the object."
+    },
+    "ods:type": {
+      "type": "string",
+      "examples": ["https://doi.org/21.T11148/2037de437c80264ccbce"],
+      "description": "This is the handle of the type of the object."
+    },
+    "ods:version": {
+      "type": "integer",
+      "examples": [1],
+      "description": "The version of the object."
+    },
+    "dcterms:creator": {
+      "type": "object",
+      "properties": {
+        "ods:type": {
+          "type": "string",
+          "examples": ["foaf:Person"],
+          "description": "Object containing information on who created the object.",
+          "enum": [
+            "foaf:Person",
+            "foaf:Organisation",
+            "oa:SoftwareAgent"
+          ]
+        },
+        "foaf:name": {
+          "type": "string",
+          "examples": ["My Name"],
+          "description": "Name of the creator."
+        }
+      },
+      "description": "The creator of the collection."
+    },
+    "ltc:ObjectGroup": {
+      "type": "object",
+      "properties": {
+        "ltc:conditionsOfAccess": {
+          "type": "boolean",
+          "examples": ["Open to the public"],
+          "description": "Information about who can access the collection being described or an indication of its security status."
+        },
+        "ltc:collectionName": {
+          "type": "string",
+          "examples": ["Arctiacaja1991Netherlands"],
+          "description": "A short title that summarizes the collection objects contained within the ObjectGroup."
+        },
+        "ltc:description": {
+          "type": "string",
+          "examples": ["A virtual collection representing all butterfly specimens in South Holland"],
+          "description": "A free text description or narrative about the collection."
+        },
+        "ltc:isKnownToContainTypes": {
+          "type": "array",
+          "description": "Flag property to indicate that the collection is known to include type specimens. `true` - this collection contains types; `false` - this collections does not contain types.",
```

Comment:
Why is this an array of booleans? 
This might be a costly flag to check. Remember, virtual collections are like saved searches, so they're dynamic. If we have a VC with 10 million items, are we supposed to check regularly if the collection contains a type specimen? 
@samleeflang thoughts?

Reply:
yes, it could be costly to check. But from the use case side, I see value. As a user, I want to find collections that has type specimens.

Reply:
I don't think it is possible to set this and keep it in sync. What if new specimen come in or specimen are updated, we would have to recheck tens, hundreds, thousands? of virtual collections to see if this need an update as well... Might be best not to include this property.

Reply:
Same for the count as well. We shouldn't put it in the object, but it should be calculated when it is displayed (so the filters are run). We can keep the fields the measurementOrFact fields as they are generic enough.